### PR TITLE
Draft for v2 release notes

### LINKED ISSUE #52: Add annoncement article for 2.0 release
1. Add change log
1. Add note that the project is now community driven. Mention @silverhook 
1. Make special mention of @iranzo and @AWegnerGitHub 
1. Add thanks https://github.com/Pelican-Elegant/elegant/issues/227

### PR DESCRIPTION
**Pull request by iranzo**
Closes #52

### CODE REVIEW

File: content/Elegant - Pelican Theme/v2.md [hunk 1]
```diff
@@ -0,0 +1,332 @@
+---
+Title: Elegant 2.0 release notes
+Tags: elegant, v2, release
+layout: post
+date: 2019-01-03 09:00
+comments: false
+category:
+slug: v2release
+---
```
Comment (silverhook) [inline content/Elegant - Pelican Theme/v2.md]: How about adding a summary?

```
Summary: Big improvements on all fronts – loads of bug fixes, improved W3C conformance, a community development model, and a new website.
```

File: content/Elegant - Pelican Theme/v2.md [hunk 2]
```diff
@@ -0,0 +1,332 @@
...
+
+# Elegant v 2.0 is finally here!
+
+A lot has happened since the last release of elegant theme and a great effort was also done and new steps for the future have been set as the way forward.
+
+V2 is not only a new set of bytes but also features and probably, the most important step has been evolving elegant into a community effort.
+
+Tasha, who created the elegant theme did helped this movement, setting a new home for the repository and the documentation plus adding some next steps towards evolving  the theme faster in the future by removing some dependencies.
```
Comment (silverhook) [inline content/Elegant - Pelican Theme/v2.md]: s/Tasha/Talha

**Comment by silverhook** [comment]
Please find below my draft (minus the actual ChangeLog) that I wrote in parallel. @iranzo, feel free to copy what you think is useful.

I have left some comments in-line in your draft as well.

```
Title: Elegant 2.0 release notes
Tags: pelican-theme, web-design, project-management
Category: Release notes
Date: 2018-12-27 20:00
Slug: elegant-2-0-release-notes
Subtitle: First community-ran release
Summary: Big improvements on all fronts – loads of bug fixes, improved W3C conformance, a community development model, and a new website.
Keywords:

[TOC]

# Introduction

With more than 4 years in the making, this release started “better late then never”, but it turned out to _so much **much**_ more then that.

With the [community spark re-ignited][announcement_community], this has become the biggest release since probably 1.0. And as such a very worthy release to carry the 2.0 crown.

[announcement_community]: {filename}/community-drien-project.md

# Stats

**119 issues** were [closed in the 2.0 release][milestone-2.0] – an impressive number, even if we take into account that many of the bugs were of an organisational nature, as Pelican Elegant has changed the development and governance model (more on that in a [separate post][announcement_community]). Compare that to [1.3 release][milestone-1.3], which consisted of 4 issues, or the total amount of closed issues so far, which amount to 133.

[milestone-2.0]: https://github.com/Pelican-Elegant/elegant/milestone/3?closed=1
[milestone-1.3]: https://github.com/Pelican-Elegant/elegant/milestone/1?closed=1

Up until [1.3 release][contrib_to-1.3], the only committer was [Talha Mansoor][] with 357 commits.

From 1.3 release until the [2.0 release][contrib_to-2.0] there were **302 new commits in total** and divided as follows:

- [Talha Mansoor][]: 264 commits
- [Pablo Iranzo Gómez][]: 7 commits
- [Calf Zhou][]: 7 commits
- [Andrew Wegner][]: 6 commits
- [Matija Šuklje][]: 5 commits
- [Jeremy Thurgood][]: 1 commit
- [“Mobile Developer”][]: 1 commit
- [Leo Torres][]: 1 commit
- [Gan Shen][]: 1 commit
- [Gert van Dijk][]: 1 commit
- [Miguel Lechón][]: 1 commit

[contrib_to-1.3]: https://github.com/Pelican-Elegant/elegant/graphs/contributors?to=2013-10-11&type=c
[contrib_to-2.0]: https://github.com/Pelican-Elegant/elegant/graphs/contributors?from=2013-10-12&to=2018-12-27&type=c

As we can clearly see, by any metric this is a huge milestone for Elegant.

[pelican]: https://getpelican.com
[Andrew Wegner]: https://andrewwegner.com
[Ashwin Vishnu Mohanan]: https://ashwinvis.github.io/
[Calf Zhou]: http://gocalf.com
[Talha Mansoor]: http://oncrashreboot.com
[Pablo Iranzo Gómez]: https://iranzo.github.io/
[Matija Šuklje]: https://matija.suklje.name
[Jeremy Thurgood]: http://rhetoric.jerith.org/
[“Mobile Developer”]: https://0x8badfood.github.io/blog/
[Leo Torres]: http://leotrs.com/
[Gan Shen]: https://gshen42.github.io/
[Gert van Dijk]: https://blog.g3rt.nl/
[Miguel Lechón]: https://blog.debiatan.net/

# Highlights

Most issues belonged to bugs and dependency updates, amongst the biggest:

- support for HTTPS out of the box by making the links (incl. CDN) protocol agnostic
- fix for search to work again
- fix of accordion menus not opening up – fixes both issues with categories and comments
- fix of table of content
- much improved build speed
- support for Jinja 2.9 (and newer)

But also new features were added. To list just a few:

- article summaries in recent posts
- added links to social networks (if so desired)
- support for several analytics providers
- support for the `series` plugin (instead of the deprecated `multi_part` plugin)
- non-English languages are now possible as default, as well as having translations of articles
- big steps towards full W3C compliance
- support of LaTeX as input format
- support for Disqus comments
- new website and documentation (more on that in a [separate post][announcement_community])

For a full changelog, see below.


# Full ChangeLog

Below is the full changelog:

## Version 2.0 (under development)
```

File: content/Elegant - Pelican Theme/v2.md (hunk 1 above)
Comment (iranzo) [reply content/Elegant - Pelican Theme/v2.md]: Added in next upload

File: content/Elegant - Pelican Theme/v2.md (hunk 2 above)
Comment (iranzo) [reply content/Elegant - Pelican Theme/v2.md]: Done in next release

**Comment by iranzo** [comment]
Added your suggestions + changes in your release notes, needs to merge with your file 'community-driven-development'

**Comment by silverhook** [comment]
LGTM. @talha131 ?

**Comment by silverhook** [comment]
> needs to merge with your file 'community-driven-development'

in https://github.com/Pelican-Elegant/documentation/issues/52#issuecomment-450228727 Talha made a good point to have the community dev model as a separate announcement. That’s why it’s in a separate file.

I think the best course of action would be to accept @iranzo’s PR here and that I remove the 2.0 release notes from my PR and leave just the community announcement there.

**Comment by iranzo** [comment]
I meant to 'have the file in your PR' with the 'merge', as this release notes also have the link you included in yours to that additional file.

The plan you provided is good, or if you want to submit the changes to this release file as part of your PR or ammend on this one (so that you retain autorship as you did a really good job).

As said, mines were wrote while waiting at doctor queue from the mobile phone, just to be used as a draft to start it

**Comment by silverhook** [comment]
I’ll close my PR and just push the additional file to yours.

**Comment by iranzo** [comment]
Great, let's have @talha131 and @AWegnerGitHub and @calfzhou doing review on this for merging

**Comment by ashwinvis** [comment]
LGTM!

**Comment by talha131** [comment]
LGTM. But I suggest we add these articles as draft for now.

https://docs.getpelican.com/en/stable/content.html?highlight=draft#publishing-drafts

We will remove the draft status when we do the actual release. What do you say?

**Comment by iranzo** [comment]
hopefully 'done'

**Pull request merged**